packages/client/src/components/Tabs/index.js

```diff
@@ -66,34 +103,32 @@ export default function VTabs(props) {
 
     return (
         <div className={classes.root}>
-            <Tabs
+            <ProcessTabs
                 /* sets horizontal or vertical tabs */
                 orientation="vertical"
                 /* changes type of tab feature if tabs > height*/
                 // variant="scrollable"
```

Comment:
let's delete this comment

Reply:
which comment?

Reply:
`                // variant="scrollable"`